prov/efa/src/dgram/efa_dgram_ep.c

```diff
@@ -444,16 +442,6 @@ int efa_dgram_ep_open(struct fid_domain *domain_fid, struct fi_info *user_info,
 	 */
 	assert(user_info->tx_attr->iov_limit <= 2);
 
-	ret = ofi_bufpool_create(&ep->send_wr_pool,
```

Comment:
I thought this should be part of your earlier change that implements the msg interface? Am I wrong?

Reply:
Yes I realized I didn't include it in the last PR

Reply:
I did. Do you mean separate PRs?

Reply:
Sorry, I missed that it's already in separate commit. No bother then